Source/Core/InputCommon/ControllerEmu/ControlGroup/Force.cpp

```diff
@@ -26,26 +25,68 @@ Force::Force(const std::string& name_) : ControlGroup(name_, GroupType::Force)
   controls.emplace_back(std::make_unique<Input>(Translate, _trans("Forward")));
   controls.emplace_back(std::make_unique<Input>(Translate, _trans("Backward")));
 
-  numeric_settings.emplace_back(std::make_unique<NumericSetting>(_trans("Dead Zone"), 0, 0, 50));
+  // Maximum swing movement (centimeters).
+  numeric_settings.emplace_back(std::make_unique<NumericSetting>(_trans("Distance"), 0.25, 1, 100));
+
+  // Maximum jerk (m/s^3).
+  numeric_settings.emplace_back(std::make_unique<NumericSetting>(_trans("Jerk"), 5.0, 1, 1000));
```

Comment:
Please put an i18n comment for this string that explains what kind of jerk this is (the derivative of the acceleration). It's easy for translators to misinterpret this otherwise.

Reply:
👍